test/image/mocks/legendgroup_horizontal_wrapping.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
For comparison, mock looks like: 

![image](https://user-images.githubusercontent.com/6675409/54542461-32ddb200-4972-11e9-9ea6-fd529f83db44.png)


on master.